S100-Shop-Facade-Open-Api-Sepc-v0.1.yaml

```diff
@@ -0,0 +1,1145 @@
+                - title: Port Radar
+                  upn: 1A1DAD797C966EC9F6A5500000000000001111111
+                - title: Starboard Radar
+                  upn: 1A1PAD797C966EC9F6A5500000000000001111111
+        "204":
+          description: No Content - There are no UPNs for the licence.
+          headers:
+            X-Correlation-ID:
+              description: GUID for the request for logging/tracing
+              schema:
+                type: string
+          content: {}
+        "400":
+          description: Bad request - could be missing or invalid licenceId, it must be an integer and greater than zero.
+          headers:
+            X-Correlation-ID:
+              description: GUID for the request for logging/tracing
+              schema:
+                type: string
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/errorDescription'
+              example:
+                correlationId: 6c254109-8a0f-4446-8649-721775ba9de2
+                errors:
+                - source: licenceId
+                  description: Invalid LicenceId.
+        "401":
+          description: Unauthorised - either you have not provided valid token, or your token is not recognised.
+          headers:
+            X-Correlation-ID:
+              description: GUID for the request for logging/tracing
+              schema:
+                type: string
+          content: {}
+        "403":
+          description: Forbidden - you have no permission to use this API.
+          headers:
+            X-Correlation-ID:
+              description: GUID for the request for logging/tracing
+              schema:
+                type: string
+        "404":
+          description: Licence not found.
+          headers:
+            X-Correlation-ID:
+              description: GUID for the request for logging/tracing
+              schema:
+                type: string
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/errorDescription'
+              example:
+                correlationId: 6c254109-8a0f-4446-8649-721775ba9de2
+                errors:
+                - source: licenceId
+                  description: Licence Not Found.
+        "429":
+          description: Too Many Requests.
+          headers:
+            X-Correlation-ID:
+              description: GUID for the request for logging/tracing
+              schema:
+                type: string
+        "500":
+          description: Internal Server Error.
+          headers:
+            X-Correlation-ID:
+              description: GUID for the request for logging/tracing
+              schema:
+                type: string
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/exceptionDescription'
+                
+  /holdings/{licenceId}/s100:
```

Comment:
We should try and be consistent in the paths and have either {licenceId} first, or s100 first, when compared to the `getPermits` and `getUserPermits` operation.  My feeling is that both should have the licenceId first (so this one is correct).

Reply:
Ok.